src/test/java/com/cloudbees/jenkins/support/impl/CustomLogsTest.java

```diff
@@ -77,31 +76,18 @@ public void testCustomLogRotation() throws IOException {
         customLogsDir.mkdirs();
 
         // Create dummy log for test1 log recorder
-        Files.write(Paths.get(customLogsDir.getPath(), "test1.log"), "test1 one".getBytes(), StandardOpenOption.CREATE);
-        Files.write(
-                Paths.get(customLogsDir.getPath(), "test1.log.1"), "test1 two".getBytes(), StandardOpenOption.CREATE);
-        Files.write(
-                Paths.get(customLogsDir.getPath(), "test1.log.2"), "test1 three".getBytes(), StandardOpenOption.CREATE);
+        Files.write(Paths.get(customLogsDir.getPath(), "test1.log"), "test1 one".getBytes());
```

Comment:
You can just use https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/nio/file/Files.html#writeString(java.nio.file.Path,java.lang.CharSequence,java.nio.file.OpenOption...)